apps/web/src/components/layout/components/SideNav.tsx

```diff
@@ -76,7 +76,7 @@ export function SideNav({}: Props) {
 
   const menuItems = [
     {
-      condition: !readonly && currentUser?.showOnBoarding,
+      condition: !readonly,
```

Comment:
getting started page navigation item will be always visible (that's what Nikolay wanted)